package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "simplerecipe",
+  "version": "1.0.0",
```

Comment:
Not sure how important this is atm but this should probably be 0.0.0.1 or something.

Reply:
Yeah will change this.